[DO NOT REVIEW] Az File Share - Add az powershell auto generation fro…

ankushbindlish2 opened the pull request:
…m typespec

# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

<a href="?expand=1&template=sdk_configuration_template.md">Click here</a> to open a PR for only SDK configuration.

ankushbindlish2 committed:
[DO NOT REVIEW] Az File Share - Add az powershell auto generation from typespec

josefree commented:
The service is under folder structure migration.  Please ensure to contact azversioning@service.microsoft.com for more details.  When you reach out, ensure to bring the pull request you want to make progress.

ankushbindlish2 committed:
Add v6 resource management common type definitions

Copied from v5. Introduces new JSON schema files for customer managed keys, managed identity with delegation, managed-on-behalf-of configuration, and network security perimeter. These definitions provide reusable common types for resource providers to standardize API specifications.
